api/turing/imagebuilder/imagebuilder.go

```diff
@@ -43,7 +43,12 @@ const (
 type JobStatus int
 
 const (
-	tickDurationInSeconds = 5
+	// jobDeletionTimeoutInSeconds is the maximum time to wait for a job to be deleted from a cluster
+	jobDeletionTimeoutInSeconds = 10
```

Comment:
Same comment as https://github.com/caraml-dev/merlin/pull/411/files#r1233458341

Shall we make it 30 seconds to give sufficient time?